Test_reviewnb.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>Here are some suggestions for changes to the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acfogarty/test-reviewnb/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>